library/core/src/slice/mod.rs

```diff
@@ -73,6 +73,9 @@ pub use sort::heapsort;
 #[stable(feature = "slice_get_slice", since = "1.28.0")]
 pub use index::SliceIndex;
 
+#[unstable(feature = "inherent_escape_default", issue = "77174")]
```

Comment:
I think this should also be `inherent_ascii_escape`:

```suggestion
#[unstable(feature = "inherent_ascii_escape", issue = "77174")]
```